src/BrokerClient.ts

```diff
@@ -0,0 +1,254 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { BaseRestClient } from './lib/BaseRestClient.js';
+import {
+  REST_CLIENT_TYPE_ENUM,
+  RestClientOptions,
+  RestClientType,
+} from './lib/requestUtils.js';
+import {
+  BrokerTransferRequest,
+  CreateBrokerSubAccountApiRequest,
+  DeleteBrokerSubAccountApiRequest,
+  GetBrokerDepositListRequest,
+  GetBrokerInfoRequest,
+  GetBrokerSubAccountApisRequest,
+  GetBrokerSubAccountsRequest,
+  UpdateBrokerSubAccountApiRequest,
+} from './types/request/broker.types.js';
+import {
+  BrokerDepositRecord,
+  BrokerInfo,
+  BrokerSubAccountApi,
+  BrokerTransferHistory,
+  BrokerWithdrawalRecord,
+  CreateBrokerSubAccountApiResponse,
+  CreateBrokerSubAccountResponse,
+  GetBrokerSubAccountsResponse,
+} from './types/response/broker.types.js';
+import { APISuccessResponse } from './types/response/shared.types.js';
+
+/**
+ *
+ */
+export class BrokerClient extends BaseRestClient {
+  constructor(
+    restClientOptions: RestClientOptions = {},
+    requestOptions: AxiosRequestConfig = {},
+  ) {
+    super(restClientOptions, requestOptions);
+    return this;
+  }
```

Comment:
```suggestion
```

Btw, since the constructor isn't actually doing anything - it's just calling the extended class, you can remove it. I know I did the same in the other classes - not sure why I didn't remove that either.